use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -253,6 +269,15 @@ following instructions with kernel `v4.19.28-41`.
 $ mkdir -p $GOPATH
 $ cd $GOPATH
 $ go get -v github.com/kata-containers/packaging
+$ cat << EOF > $GOPATH/src/github.com/kata-containers/packaging/kernel/configs/fragments/common/qat.conf
+CONFIG_MODULES=y
```

Comment:
We already know `CONFIG_MODULES` and `CONFIG_STACKPROTECTOR` will be set as defaults to perhaps leave them out here

Reply:
I removed all 3 optional sections that deal with insecure modules, installing LTS kernel, and compiling the QAT driver against the host.  IIRC the LTS kernel was only needed to build the 4.6 driver for installing to the host because of issues with compiling with the 5.x kernel.  None of those sections are needed if using the Clear Linux QAT driver and firmware bundle.

Reply:
I will remove those two modules so I don't have to update the instructions again later.